tests/test_optimization/test_trigger1_conflict_detection.py

```
@@ -0,0 +1,890 @@
+             patch(_MODEL_PATCH, return_value="gemini/gemini-2.5-flash"), \
+             patch(_KWARGS_PATCH, return_value={}):
+
+            _fire_llm_conflict_detection(
+                feedback_loop=feedback_loop,
+                workflow_id="wf-partial-flag",
+                failed_code="v1", working_code="v2",
+                url="https://demoqa.com",
+            )
+
+        flagged_ids = feedback_loop.nl_engine.conflict_flag_hints.call_args[0][0]
+        assert 22 not in flagged_ids, f"Hint 22 should not have been flagged"
```

Ruff (0.15.14)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)